assets/src/edit-story/components/panels/link/index.js

```diff
@@ -150,73 +154,61 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
     [handleChange]
   );
 
-  // Informational dialog
-  const [infoDialogOpen, setInfoDialogOpen] = useState(false);
-  const openDialog = useCallback(() => setInfoDialogOpen(true), []);
-  const closeDialog = useCallback(() => setInfoDialogOpen(false), []);
-
   return (
     <SimplePanel name="link" title={__('Link', 'web-stories')}>
       <Row>
-        <Note onClick={() => openDialog()}>
-          {__('Type an address to apply a 1 or 2-tap link', 'web-stories')}
-        </Note>
+        <Note>{__('Type an address to apply a link', 'web-stories')}</Note>
       </Row>
 
       <Row>
         <ExpandedTextInput
           placeholder={__('Web address', 'web-stories')}
           disabled={!canLink}
-          onChange={(value) => handleChange({ url: value })}
+          onChange={(value) =>
+            handleChange({ url: value }, !value /* submit */)
+          }
           value={link.url || ''}
           clear
         />
       </Row>
 
-      {Boolean(link.url) && link.type === LinkType.TWO_TAP && (
+      {Boolean(link.url) && (
         <Row>
           <ExpandedTextInput
             placeholder={__('Optional description', 'web-stories')}
             disabled={!canLink}
-            onChange={(value) => handleChange({ desc: value })}
+            onChange={(value) =>
+              handleChange({ desc: value }, !value /* submit */)
```

Comment:
Hmm. Do we really need this for `desc`?

Reply:
I think it makes sense to see the description cleaned immediately, no?

Reply:
Yeah. I think you're right.